config/routes.rb

```diff
@@ -5,6 +5,7 @@
 
     resources :games, only: [:index]
     resources :user_games, only: [:create]
+    resources :results, only: [:index]
```

Comment:
I was thinking this might be namespace as quiz/results
Feel free to change it if that makes more sense.

Reply:
good thought, I dont have any strong feelings at the moment